Makefile.tools

```diff
@@ -20,7 +20,7 @@ OPM=$(TOOLS_DIR)/opm
 OPM_VERSION = v1.15.1
 
 GOLANGCI_LINT=$(TOOLS_DIR)/golangci-lint
-GOLANGCI_LINT_VERSION = v1.42.1
+GOLANGCI_LINT_VERSION = v1.45.2
```

Comment:
it seems like we have missed adding the tool to the cache file - ` .github/tools`
I will submit a patch.